sysdump/constants.go

```diff
@@ -51,6 +52,7 @@ const (
 	ciliumSPIREAgentConfigMapFileName        = "cilium-spire-agent-configmap-<ts>.yaml"
 	ciliumSPIREServerStatefulSetFileName     = "cilium-spire-server-statefulset-<ts>.yaml"
 	ciliumSPIREServerConfigMapFileName       = "cilium-spire-server-configmap-<ts>.yaml"
+	ciliumSPIREServerEntriesFileName         = "cilium-spire-server-entries-%s-<ts>.log"
```

Comment:
file ending `log` seems not to fit here.

what do you think of changing the output format to json (by adding `--output json` to the command) and changing the file ending accordingly? Should also be easier to process and analyze the file.

Reply:
Good one! Didn't think it would have JSON output, I changed it!